quickstarts/consolequickstart-install-cryostat.yaml

```diff
@@ -0,0 +1,146 @@
+        
+        1. Click **View installed Operators in Namespace** to open the **Installed Operators** page.
+
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again, or [read
+          more](https://access.redhat.com/documentation/en-us/red_hat_build_of_cryostat/2/html/installing_cryostat/assembly_installing-cryostat_cryostat)
+          about this topic.
+        instructions: >
+          #### Verify that the Red Hat build of Cryostat is installed:
+          
+          - From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+
+          - The Red Hat build of Cryostat should be shown in the list of Operators.
+
+          - In the **Status** column, do you see the status of the Red Hat build of Cryostat as `Succeeded`?
+    - title: Create a Cryostat instance
+      description: >
+        To create a Cryostat instance:
+
+        1. From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+        
+        1. In the **Search by name** field type and search for the `Red Hat build of Cryostat`. If the operator is not found, ensure that you have selected the project where the Cryostat Operator was installed. 
+
+        1. Click **Red Hat build of Cryostat** to open the Operator details.
+
+        1. Click the **Cryostat** tab.
+
+        1. Click **Create Cryostat**. A form with default values opens.
+       
+        1. Specify a new name for the Cryostat instance. 
+        
+          **Note:** No other updates are required in the form to proceed. However, based on your requirements, you can also update the optional fields on the form. 
+       
+        1. Click **Create**.
+
+      review:
+        failedTaskHelp: >-
+          This task isn’t verified yet. Try the task again, or [read
+          more](https://access.redhat.com/documentation/en-us/red_hat_build_of_cryostat/2/html/installing_cryostat/assembly_installing-cryostat_cryostat)
+          about this topic.
+        instructions: >
+          #### Verify that your Cryostat instance has been created:
+
+          From the **Administrator** perspective, go to **Installed Operators** from the [Operators]{{highlight qs-nav-operators}} section of the navigation. 
+
+          - Go to the Red Hat build of Cryostat operator and click the **Cryostat** tab.
+
+          - Do you see the Cryostat instance in the Cryostats list?
+
+          - Click the Cryostat instance in the **Cryostats** list. 
+
+            - You should see a URL to the Cryostat application in Application URL.
+            - In the **Cryostat Conditions** table, do you see the **TLSSetupComplete** and **MainDeploymentAvailable** types set to `True`?
```

Comment:
These two sub-list items aren't shown as sublist. Is it possible:

![image](https://github.com/openshift/console-operator/assets/139310/f8cb0b11-4f00-4e03-80c5-64ddbec24a43)

You need two more spaces per item:

```suggestion
              - You should see a URL to the Cryostat application in Application URL.
              - In the **Cryostat Conditions** table, do you see the **TLSSetupComplete** and **MainDeploymentAvailable** types set to `True`?
```